### ISSUE TITLE
Too many "WARN DTLSConnector - Cleanup 0 recent handshakes, left 1" ? 

### ISSUE THREAD

**Issue by sbernard31** [eclipse/californium#1668]
(Californium 3.0.0-M2)

I get some logs : 
```
2021-07-15 15:45:27,027 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:45:32,028 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:45:37,028 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:45:42,029 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:45:47,030 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:45:52,030 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:45:57,031 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:46:02,032 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:46:07,033 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:46:12,034 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:46:17,035 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:46:22,036 WARN DTLSConnector - Cleanup 0 recent handshakes, left 1!
2021-07-15 15:46:27,036 WARN DTLSConnector - Cleanup 1 recent handshakes, left 0!
```

I'm not sure why ? I just do 1 failing handshake and get all those logs. 

I'm not sure if this expected but it seems very verbose for a WARN log. 

(I need to finish something first then I can try to investigate more if needed)

**Comment by boaks** [comment]
"Recent handshakes" are the kept "client random"s in order to filter "unexpected retransmissions" of CLIENT_HELLOs. The current master already adjusted it to info, I plan to use debug.

**Comment by boaks** [comment]
Fixed on master with 48b5beb48a2b9a22e5fff98e13c3e772ccc69932
